Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -34,11 +34,62 @@ public extension Array {
         swapAt(index, otherIndex)
     }
 }
+// MARK: - Methods (Hashable)
+public extension Array where Element: Hashable {
+    /// SwifterSwift: Remove all instances contained in items parameter from array. Complexity: O(n)
```

Comment:
Nit: Instead of adding complexity here, we could add it as the key so it would be highlighted on the docs generated.
Example: 
```
/// - Parameter items: unique items to remove.
/// - Returns: self after removing all instances of all items in given array.  
/// - Complexity: O(1)

```